@xen-orchestra/fs/src/s3.js

```diff
@@ -31,6 +32,11 @@ export default class S3Handler extends RemoteHandlerAbstract {
     if (protocol === 'http') {
       params.httpOptions.agent = new http.Agent()
       params.sslEnabled = false
+    } else if (protocol === 'https') {
+      params.httpOptions.agent = new https.Agent({
+        rejectUnauthorized: !allowUnauthorized,
+        keepAlive: true,
```

Comment:
Why `keepAlive`?

Reply:
it keeps the socket opens during a grace period (by default 1s), it speeds up notably multiple https queries since tls negocation is costly